### DISCUSSION TITLE
Must vehicle be enabled first?

### DISCUSSION THREAD

**Discussion by ucmmesa** [openTCS/opentcs#45] [Q&A]
Hi!

I use openTCS 5.6.
There are two vehicles in my plant model:vehicle-001 and vehicle-002.
If I change the integration level of the vehicle-002 to be_utilized and set the sending point,  and I could see the colored path showing the route to the move order by name Move-xxxxxxx,and then I check the loopback adapter of vehicle-002 in the control center,and finally enable the vehicle.However,the move order keeps the status of being_processed and there are no animation in the operationas desk.
![image](https://user-images.githubusercontent.com/11491673/189026416-2c6dc5b8-53b1-4275-8eff-82765029387b.png)

Should the vehicle enable be set first and then to generate transport order?

**Comment by swltr** [comment]
No, you should not need to enable the vehicle first, and I cannot reproduce that behaviour based on your description. Maybe there was something else you did to get in to that situation?

**Reply by jfaigan** [reply]
I had a similar problem with my small test model.  Upon creation of the Transport Order, I would see it highlighted, the order would be associated to my vehicle and it would be in the status being_processed, however, the vehicle was staying as IDLE.

I loaded the demo model and tried to create a transport order and this one worked.  I saw the route highlighted and then I saw the vehicle jumping between points. 

I am going to investigate what is different between the more complex demo model and my simple model.

I will post back if I find anything.

Regards,
Joel

**Reply by jfaigan** [reply]
Ok, I found the problem with my simple model.  I had created some Peripheral elements that seemed to be incorrect.  I removed all peripheral elements and republished to the kernel and I was able to get it working.

Hope this helps.
